rfcs/0132-use-meson-to-build-nix.md

```diff
@@ -0,0 +1,121 @@
+---
+feature: use_meson_to_build_nix
+start-date: 2022-08-25
+author: Anderson Torres
+co-authors: @p01arst0rm
+shepherd-team:
+shepherd-leader:
+related-issues:
+---
+
+# Summary
+[summary]: #summary
+
+Use meson as an alternative build system for the reference implementation of
+Nix.
+
+# Motivation
+[motivation]: #motivation
+
+Currently, the reference implementation of Nix evaluator and its companion
+toolset generated from the Nix source tree are built using an autotools-like
+script.
```

Comment:
I think it is good to be precise
```suggestion
toolset generated from the Nix source tree are built using a configure script made with Autoconf, and GNU Make.
```
Also I would not bother trying to wrap on line length because it makes editting hard. (I do line breaks on punctuation.)